finetune/comparison.py

```diff
@@ -29,8 +29,8 @@ def _text_to_ids(self, pair, Y=None, pad_token=None):
 
     def feed_shape_type_def(self):
         TS = tf.TensorShape
-        return ({"tokens": tf.int32, "mask": tf.int32}, tf.int32), (
-            {"tokens": TS([2, self.config.max_length, 2]), "mask": TS([2, self.config.max_length])},
+        return ({"tokens": tf.int32, "mask": tf.float32}, tf.float32), (
+            {"tokens": TS([None, self.config.max_length, 2]), "mask": TS([None, self.config.max_length])},
```

Comment:
Do we support comparison w/ > 2 items?  Curious why this change was necessary?

Reply:
I don't believe this is actually necessary. I changed it to try to trick the shape inference of datasets.